docs/server/source/data-models/block-model.rst

```diff
@@ -27,7 +27,7 @@ A block has the following structure:
       It's the list of federation nodes which can cast a vote on this block.
       This list can change from block to block, as nodes join and leave the federation.
 
-- ``signature``: Cryptographic signature of the block by the node that created the block. (To create the signature, the node serializes the block contents and signs it with its private key.)
+- ``signature``: :ref:`Cryptographic signature <Signature Algorithm and Keys>` of the block by the node that created the block. To generate the signature, the node builds a dict including the ``id``, the inner ``block`` & the ``signature`` (with a value of ``None``), it serializes that dict, and then signs *that* with its private key.
```

Comment:
> signature of the block by the node that created the block (`node_pubkey`)

It may be worth to say that the block needs to be signed by `node_pubkey` 

> To generate the signature, the node builds a dict including the ``id``, the inner ``block`` & the ``signature`` (with a value of ``None``)

This is wrong. The signature is only done on the inner block (`timestamp`, `transactions`, `node_pubkey`, `voters`) https://github.com/bigchaindb/bigchaindb/blob/ae029c7bcfebbe8cfcb2058a642c5ddddcf641b1/bigchaindb/models.py#L281